automate-gitops-pull-request.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# Please Use Google Shell Style: https://google.github.io/styleguide/shell.xml
+
+# ---- Start unofficial bash strict mode boilerplate
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -o errexit  # always exit on error
+set -o errtrace # trap errors in functions as well
+set -o pipefail # don't ignore exit codes when piping output
+set -o posix    # more strict failures in subshells
+# set -x          # enable debugging
+
+IFS=$'\n\t'
+# ---- End unofficial bash strict mode boilerplate
+
+error=""
+for VAR in SERVICE DOCKER_REPOSITORY GITHUB_TOKEN GH_USERNAME GH_EMAIL REACTION_GITOPS_REVIEWERS; do
+  if [[ -z "${!VAR}" ]]; then
+    error="${error}Missing required environment variable: ${VAR}\n"
+  fi
+done
+if [[ -n "${error}" ]]; then
+  echo -e "${error}" 1>&2
+  exit 1
+fi
+
+# Install Kustomize
```

Comment:
Normally this script will NOT be run from the `ci-scripts` directory so you'll need to compute the full path to that the way we do [here](https://github.com/reactioncommerce/ci-scripts/blob/c083738d73a3894b83b9e84c0c315d21116461b4/docker-build-tag-push.sh#L26) then run the sibling scripts with absolute paths via that variable.

Reply:
updated to ```"${ci_scripts_dir}"/install-kustomize.sh```